### DISCUSSION TITLE
Is there a way to print() from  withing SuspendCurses?

### DISCUSSION THREAD

**Discussion by frosencrantz** [saulpw/visidata#1485] [Q&A] [answered]
I'm using the latest develop version of Visidata, and noticed that I can no longer use the `print()` function from within a `SuspendCurses`.    What do you recommend for doing this now?  I have some custom code that will print out something to the screen.

**Comment by geekscrapy** [comment]
This _should_ work, I am doing it.
Do you have a minimal example?

**Reply by frosencrantz** [reply]
Hmm...  I get different results in different configurations, but here is an example using the builtin REPL.  The print function output goes to the Status sheet not to stdout.

https://asciinema.org/a/XaDaBTsYrUOroUe5MvlIAdprK

**Comment by saulpw** [comment, marked as answer]
We just changed this in develop, I didn't know that people were still using print() builtin within VisiData.  You can use `vd.printout()` to stdout and `vd.printerr()` to stderr.  We'll probably make SuspendCurses swap in the builtin and swap it out again when it completes.

**Reply by frosencrantz** [reply]
vd.printout() works for my case.  I still use print with SuspendCurses.  I want to print something out to the screen, so I have some control over what I copy/paste.  Thanks, @saulpw!  And thanks @geekscrapy for checking.
